katana/wui/core/views.py

```diff
@@ -89,3 +106,245 @@ def check_if_file_exists(request):
     else:
         output = {"exists": file_or_dir_exists(join_path(directory, filename + extension))}
     return JsonResponse(output)
+
+
+
+
+
+class UserAuthView(View):
+    """
+    User authentication view
+    """    
+    
```

Comment:
Hi @satyakrish,

The functionality works well. The print statements end up printing out usernames and passwords though. Can those be removed please?

Reply:
done